subworkflows/local/alignment.nf

```diff
@@ -56,20 +52,29 @@ workflow ALIGNMENT {
             }
         }
 
+        SAMTOOLS_VIEW( ch_bam_bai, ch_genome_fasta, [] )
```

Comment:
```suggestion
        SAMTOOLS_VIEW( STAR_ALIGN.out.bam.join(SAMTOOLS_INDEX.out.bai), ch_genome_fasta, [] )
```
I'm a bit torn on whether this always should be the "raw" STAR output or if we sometimes should save the depleted bam file as cram, as you wrote. I guess it depend on whether the bam file should be "complete" or "processed". What's your thoughts?